Makefile

```diff
@@ -214,8 +214,6 @@ PACKAGES_E2E=$(shell go list -tags e2e ./... | grep '/e2e')
 PACKAGES_SIM=$(shell go list ./... | grep '/tests/simulator')
 TEST_PACKAGES=./...
 
-include sims.mk
```

Comment:
asking to learn: was this never used by anyone?

Reply:
We have never used this in Osmosis